src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -211,12 +225,14 @@ Parameter MKLDNNExecNetwork::GetConfig(const std::string &name) const {
     }
 }
 
-InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name) const {
-    if (_graphs.size() == 0)
-        IE_THROW() << "No graph was found";
+Parameter MKLDNNExecNetwork::GetConfig(const std::string &name) const {
+    // because of some implementation details legacy params still need to be supported even for new API
+    return GetConfigLegacy(name);
```

Comment:
Could you please extend the comment to describe why MKLDNNExecNetwork::GetConfig differs with Engine::GetConfig. Smt about CompiledModel::get_property which try to call getMetric first and since all properties for MKLDNNExecNetwork are metrics in new API we supports here only legacy case.

Reply:
sure, will be added

Reply:
Done